packages/hash/api/src/model/ontology/entity-type.model.ts

```diff
@@ -67,11 +88,13 @@ export default class {
    *
    * @param params.ownedById - the id of the owner of the entity type
```

Comment:
```suggestion
   * @param params.ownedById - the id of the account that owns the entity type
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/1197/commits/51c3a539be68ca7bf1c7ffa8f13550e6f20efee4